src/qt/walletcontroller.cpp

```diff
@@ -356,3 +332,21 @@ void OpenWalletActivity::open(const std::string& path)
         QTimer::singleShot(0, this, &OpenWalletActivity::finish);
     });
 }
+
+LoadWalletsActivity::LoadWalletsActivity(WalletController* wallet_controller, QWidget* parent_widget)
+    : WalletControllerActivity(wallet_controller, parent_widget)
+{
+}
+
+void LoadWalletsActivity::load()
+{
+    showProgressDialog(tr("Loading wallets…"));
+
+    QTimer::singleShot(0, worker(), [this] {
```

Comment:
Use `GUIUtil::ObjectInvoke` here and below?

Reply:
I'd rather keep this PR as is, and in a follow up use `GUIUtil::ObjectInvoke` in a wider scope of all `walletcontroller.cpp`.